sys/net/gnrc/netif/Kconfig

```
@@ -12,9 +12,13 @@ menuconfig KCONFIG_MODULE_GNRC_NETIF
 
 if KCONFIG_MODULE_GNRC_NETIF
 
-config GNRC_NETIF_MSG_QUEUE_SIZE
-    int "Message queue size for network interface threads"
-    default 16
+config GNRC_NETIF_MSG_QUEUE_SIZE_EXP
+    int "Message queue size for network interface threads (as exponent of 2^n)"
```

Nit-pick: what you describe here is not the message queue size, but the expontent for the message queue size. How about (here and in all other Kconfig files changed here):

```suggestion
    int "Exponent for the message queue size for network interface threads (as 2^n)"
```

---

Changed